packages/docusaurus-plugin-pwa/src/index.js

```diff
@@ -90,9 +90,21 @@ function plugin(context, options) {
       const headTags = [];
       if (isProd && pwaHead) {
         pwaHead.forEach(({tagName, ...attributes}) => {
-          if (attributes.href && !attributes.href.startsWith(baseUrl)) {
-            attributes.href = normalizeUrl([baseUrl, attributes.href]);
-          }
+          ['href', 'content'].forEach((attribute) => {
+            const attributeValue = attributes[attribute];
+
+            if (!attributeValue) {
+              return;
+            }
+
+            const attributePath =
+              !!path.extname(attributeValue) && attributeValue;
```

Comment:
I'm not sure if really need to check for file path exists, so for now just check that attribute value contains file name.